src/libraries/System.Private.CoreLib/src/System/TimeSpan.cs

```diff
@@ -308,6 +308,149 @@ public TimeSpan Duration()
 
         public override int GetHashCode() => _ticks.GetHashCode();
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TimeSpan"/> structure to a specified number of
+        /// days.
+        /// </summary>
+        /// <param name="days">Number of days.</param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentOutOfRangeException">
+        /// The parameters specify a <see cref="TimeSpan"/> value less than <see cref="MinValue"/> or greater than <see cref="MaxValue"/>
+        /// </exception>
+        public static TimeSpan FromDays(int days) => FromDays(days, 0);
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TimeSpan"/> structure to a specified number of
+        /// days, hours, minutes, seconds, milliseconds, and microseconds.
+        /// </summary>
+        /// <param name="days">Number of days.</param>
+        /// <param name="hours">Number of hours.</param>
+        /// <param name="minutes">Number of minutes.</param>
+        /// <param name="seconds">Number of seconds.</param>
+        /// <param name="milliseconds">Number of milliseconds.</param>
+        /// <param name="microseconds">Number of microseconds.</param>
+        /// <returns></returns>
```

Comment:
> /// <returns></returns> [](http://example.com/codeflow?start=7&length=24)

looks this is missing in all added methods. I'll not comment more on that.

Reply:
I have now filled out the `/// <returns></returns>` element on all methods